src/smartpeak/source/core/RawDataProcessors/MatchSpectra.cpp

```diff
@@ -38,7 +38,7 @@ namespace SmartPeak
 
   std::vector<std::string> MatchSpectra::getRequirements() const
   {
-    return { "sequence", "traML" };
+    return { "sequence", "traML", "msp"};
   }
 
   std::set<std::string> MatchSpectra::getInputs() const
```

Comment:
Maybe MatchSpectra should have "Library" as an input

Reply:
I wanted to talk to you about this.  I am not sure how the naming for requirements vs. inputs/outputs is set-up but it seems it needs to be called "msp" for the UI to work correctly.  I tried "Library" (which I agree should be renamed) previously and I was not able to launch workflows.

Reply:
my comment is about the getInputs method. basically, at the moment the getInput/getOutput is just a matter of being able to draw the flow arrows in the workflow display. perhaps later it could have more impact.

about getRequirements it's a list of things (: filnames) that the user has to specify before launching the workflow with this processor. if the associated filename is not provided an error would be displayed before to launch it.

Reply:
getRequirements could be renamed "getFilenamesRequirement"

Reply:
> getRequirements could be renamed "getFilenamesRequirement"

Agree :)